spec/lib/web_of_science/services/hash_mappers/mapper_spec.rb

```diff
@@ -1,24 +0,0 @@
-describe WebOfScience::Mapper do
```

Comment:
I don't understand why this one commit that seems to point to something done on MapNames is also removing the spec on the mapper base class.